.github/workflows/release.sh

```diff
@@ -205,29 +239,58 @@ _releases_update() {
     if [[ "${GITHUB_REF_NAME}" == main ]]; then
         local filename_no_ext
         filename_no_ext=$(filename_no_ext_for "$1")
+
         local tar_gz_path
         tar_gz_path=$(tar_gz_path_for "$1")
 
         local crc32
         crc32=$(crc32 "${tar_gz_path}")
+
         local date
         date=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
-        echo "${filename_no_ext} ${crc32} ${date}" >>_RELEASES
-        sort -o _RELEASES _RELEASES
 
-        local release_name="release/${filename_no_ext}"
+        local _releases_created=false
+        if [[ ! -f _RELEASES ]]; then
+            echo "${filename_no_ext} ${crc32} ${date}" >_RELEASES
+            _releases_created=true
+        fi
+
         git config user.name "GitHub Actions"
         git config user.email "actions@user.noreply.github.com"
+
+        local update_releases_prefix
+        local release_name="release/${filename_no_ext}"
+        if [[ ${global_IS_NIGHTLY_OTP} == false ]]; then
+            update_releases_prefix="add"
+        else
+            if [[ ${_releases_created} == false ]]; then
+                # Replace (inline) in previously existing file (this is a special target)
+                sed -i -e "s|${filename_no_ext} \(.*\)|${filename_no_ext} ${crc32} ${date}|g" _RELEASES
+            fi
+
+            git branch -D "${release_name}"
+            git push origin --delete "${release_name}"
+            update_releases_prefix="replace"
```

Comment:
This is the major difference, the "moving target" branch. (mind you, this is untested, as of now)